include/openenclave/internal/syscall/sys/bits/epoll_event.h

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#if !defined(__aarch64__)
```

Comment:
Maybe a brief comment about how this packing it to enforce consistency with the related struct in MUSL.

Reply:
done